src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1805,8 +1805,37 @@ namespace winrt::TerminalApp::implementation
             {
                 ShellExecute(nullptr, L"open", eventArgs.Uri().c_str(), nullptr, nullptr, SW_SHOWNORMAL);
             }
+            else if (auto presenter{ _dialogPresenter.get() })
+            {
+                // FindName needs to be called first to actually load the xaml object
+                FindName(L"UnsupportedSchemeDialog").try_as<WUX::Controls::ContentDialog>();
```

Comment:
don't need the `try_as` bit since we're just throwing it away.

Alternatively: capture it as a local variable (and use `try_as` to make sure it's the right type)... because below, we use two different ways to access the same thing (`X()` and `FindName("X")`) that we already got!